docs/source/blog/derotation.md

```diff
@@ -0,0 +1,83 @@
+:blogpost: true
+:date: July 1, 2025
+:author: Laura Porta
+:location: London, UK
+:category: Blog
+:language: English
+:image: 1
+
+# `derotation`: a Python package for correcting motion artifacts in rotating multiphoton movies
+
+The `derotation` package provides a robust solution for reconstructing multiphoton movies of rotating samples acquired with a line-scanning microscope at high speeds and low frame rates. When imaging a rotating sample, the line-by-line nature of acquisition introduces significant geometric distortions. This package corrects these artifacts to produce clear, stable movies suitable for standard analysis pipelines like Suite2p.
+
+![](https://raw.githubusercontent.com/neuroinformatics-unit/derotation/main/docs/source/_static/mean_images_with_incremental.png)
+> **Left**: The mean image from a raw 3-photon movie of a rotating sample. **Center**: The mean image after processing with `derotation`. **Right**: The mean image after further registration with Suite2p. The improvement in cell definition is evident after derotation.
+
+---
+
+## Core Functionality: Line-by-Line Correction
+
+The fundamental principle of the package is **derotation-by-line**. Because a line-scanning microscope acquires an image one horizontal line at a time, a rotating sample causes each line to be captured at a slightly different angle. If the angle of rotation is recorded simultaneously, the `derotation` package can computationally rotate each line back to a common frame of reference, incrementally building a corrected, distortion-free image.
+
+![](https://raw.githubusercontent.com/neuroinformatics-unit/derotation/main/docs/source/_static/derotation_by_line.gif)
```

Comment:
I think what would made the case for line-by-line correction even stronger is to also show the derotate-by-frame correction (which doesn't work). I don't know if that will be visible with these cells. If not, you could use your nice grids.